### PR TITLE
Scenes/PanelEdit: Fix some styles and flesh out the skeleton

### PR DESCRIPTION
**Pull request by kaydelaney**
Updates styles for the options pane to better match the current styles. 
Also adds some fields (some working, some dummies) for various panel options.
Ripe for refactoring and absolutely a work in progress, but I thought it'd be better to have incremental PRs like this rather than one massive one.

### CODE REVIEW

File: public/app/features/dashboard-scene/panel-edit/PanelOptionsPane.tsx [hunk 1]
```diff
@@ -21,14 +42,86 @@ export class PanelOptionsPane extends SceneObjectBase<PanelOptionsPaneState> {
...
+            onClick={() => {
+              setVizPickerOpen(true);
+            }}
+          />
+        )}
+        <div className={styles.box}>
+          {isVizPickerOpen && (
+            <PanelVizTypePicker panelManager={panelManager} onChange={() => setVizPickerOpen(false)} />
+          )}
+          {!isVizPickerOpen && (
```
Comment (dprokop) [inline public/app/features/dashboard-scene/panel-edit/PanelOptionsPane.tsx:63]: Think we may skip this part for now and just render options placegolder before we migrate the actual options forms, wdyt?

**Review by dprokop — COMMENTED** [review]
@kaydelaney  - since you are on this already,maybe this PR could bring the styling fixes to the entire migrated panel editor? I mean fixing the horizontal paddings and making sure both bottom and right panes are looking as in the old arch?

File: public/app/features/dashboard-scene/panel-edit/PanelOptionsPane.tsx (hunk 1 above)
Comment (torkelo) [reply public/app/features/dashboard-scene/panel-edit/PanelOptionsPane.tsx:63]: yea, these needs to be done declaratively anyway (OptionsPaneItemDescriptor) to support the search

**Review by dprokop — APPROVED** [review]
Looking good, don't want to block this PR. Found two visual issues:
No left padding on the vizpanel pane and in the bottom pane tabs
image

Bottom pane does not span over the entire available space:
image

**Pull request merged**